standard/documentation-comments.md

```diff
@@ -368,10 +385,12 @@ where
 **Example:**
 
 ```csharp
-/// <permission cref="System.Security.PermissionSet">Everyone can
-/// access this method.</permission>
-public static void Test() {
-   // ...
+/// <permission cref="System.Security.PermissionSet">
```

Comment:
I saw a comment from you earlier about this, but now can't find it. Yes, let's get rid of it in a future PR - but probably get more buy-in from the other group members first.

Reply:
I created issue #472 to drive this conversion.